google/cloud/pubsub_v1/types.py

```diff
@@ -87,12 +87,14 @@
 # these settings can be altered to tweak Pub/Sub behavior.
 # The defaults should be fine for most use cases.
 FlowControl = collections.namedtuple(
-    "FlowControl", ["max_bytes", "max_messages", "max_lease_duration"]
+    "FlowControl",
+    ["max_bytes", "max_messages", "max_lease_duration", "max_extension_period"],
 )
 FlowControl.__new__.__defaults__ = (
     100 * 1024 * 1024,  # max_bytes: 100mb
     1000,  # max_messages: 1000
     1 * 60 * 60,  # max_lease_duration: 1 hour.
+    0,  # max_extension_period: disabled
```

Comment:
@IlyaFaer  Just want to know if you have an idea about why Alex hong ask to make default value -1 as it described in issue ,because value <=0  has already defined as a disabled.

Reply:
I suppose, the problem is that initially it was set to -1 in Go, as we were considering posibility to set this param to `0` for immediate retry. But after some discussion it was decided to avoid immediate retry, so we set `>=0`. And now in design doc googlers are saying that we should use `0` as default

![image](https://user-images.githubusercontent.com/26030071/75332171-00804880-5895-11ea-89b2-070aa7e1e511.png)